test/hotspot/jtreg/compiler/c2/cr6340864/TestFloatVect.java

```diff
@@ -30,6 +30,7 @@
  * @run main/othervm -Xbatch -XX:CompileCommand=exclude,*::test() -Xmx128m -XX:MaxVectorSize=8 compiler.c2.cr6340864.TestFloatVect
  * @run main/othervm -Xbatch -XX:CompileCommand=exclude,*::test() -Xmx128m -XX:MaxVectorSize=16 compiler.c2.cr6340864.TestFloatVect
  * @run main/othervm -Xbatch -XX:CompileCommand=exclude,*::test() -Xmx128m -XX:MaxVectorSize=32 compiler.c2.cr6340864.TestFloatVect
+ * @run main/othervm -Xbatch -XX:CompileCommand=exclude,*::test() -XX:TieredStopAtLevel=2 -Xmx128m -XX:MaxVectorSize=32 compiler.c2.cr6340864.TestFloatVect
```

Comment:
This question sill needs to be answered.

Reply:
Since this patch inline expands both scalar and vector implementations through C2 hence one test point which executes non intrinsic flow. But I get your point such options can be passed during jtreg invocation also.